packages/generator-electrode/generators/webapp/templates/server/plugins/webapp/react-webapp.js

```diff
@@ -72,6 +72,14 @@ function makeRouteHandler(options, userContent) {
     const renderJs = RENDER_JS && mode !== "nojs";
     const renderSs = RENDER_SS && mode !== "noss";
 
+    if (!renderSs) {
```

Comment:
I think we should keep the `noss` mode as is.  and add a new mode for this behavior. maybe `datass`